package.json

```diff
@@ -34,10 +34,13 @@
     "@semantic-release/release-notes-generator": "^10.0.3",
     "broken-link-checker-local": "^0.2.1",
     "cz-conventional-changelog": "^3.3.0",
+    "dotenv": "^16.0.0",
     "eleventy-plugin-dart-sass": "^1.0.3",
     "husky": "^7.0.4",
+    "isomorphic-fetch": "^3.0.0",
     "luxon": "^2.3.0",
-    "rimraf": "^3.0.2"
+    "rimraf": "^3.0.2",
+    "showdown": "^2.0.3"
```

Comment:
Markdown to HTML converter

Reply:
Interesting. Since 11ty converts markdown to html as well, wouldn't think this would be necessary.

Reply:
Yes, I think that eleventy is already using some convertor; But haven't investigated yet, on how to use/apply this.